build.gradle

```diff
@@ -40,3 +44,41 @@ nexusStaging {
     username = System.env['SONATYPE_USERNAME']
     password = System.env['SONATYPE_PASSWORD']
 }
+
+task changeReleaseVersion {
+    doLast {
+        def gradleFile = new File('build.gradle')
+        def gradleFileText = gradleFile.text.replaceFirst("\\nversion '\\d+\\.\\d+\\.\\d+(-.*)?'", "\nversion '" + versionParam + "'")
+        gradleFile.write(gradleFileText)
+    }
+}
+
+task changeREADMEVersion {
+    doLast {
+        def gradleFile = new File('README.md')
+        def gradleFileText = gradleFile.text.replaceFirst("com.braintreepayments.api:drop-in:\\d+\\.\\d+\\.\\d+(-.*)?'", "com.braintreepayments.api:drop-in:" + versionParam + "'")
+        gradleFile.write(gradleFileText)
+    }
+}
+
+task incrementSNAPSHOTVersion {
```

Comment:
Do we do snapshot versioning the same way as in other repos?

Reply:
This is legacy I think for the most part we stopped providing SNAPSHOT builds. On the other hand, we should consider bringing them back to allow merchants to try out unreleased features. It's probably good to have in the build script for now.